FlaskSite/utils/jwt_helper.py

```diff
@@ -0,0 +1,32 @@
+import jwt as pyjwt
+import datetime
+
+# Secret key for encoding and decoding the JWT token
+# move it to config.py 
+SECRET_KEY = "your_secret_key"
+
+
+def generate_token(user_id):
+    """
+    Generates a JWT token with the user_id as payload
```

Comment:
This Is Meaningful, But i am sharing a way I Love doing
"""
    Generates a JWT token with the user_id as payload

    Args:
        user_id (str): The unique identifier of the user.

    Returns:
        str: The encoded JWT token.
    """